src/vm/methodtablebuilder.cpp

```diff
@@ -1484,6 +1484,23 @@ MethodTableBuilder::BuildMethodTableThrowing(
         }
     }
 
+#if defined(_TARGET_X86_) || defined(_TARGET_AMD64_) 
+    // All the funtions in System.Runtime.Intrinsics.X86 are hardware intrinsics.
```

Comment:
This should be done for system module only like the other check. No need to do it for non-system module.

Reply:
Thank you! Will fix.